en/docs/tutorials/construct-statements.md

```diff
@@ -0,0 +1,119 @@
+# Construct Statements
+
+The Statement Editor in the Choreo Web Editor is a tool that allows you to construct Ballerina statements in your Choreo Editor even without an in-depth knowledge of the Ballerina Programming Language.
```

Comment:
We seem to be using the terms **Choreo Web Editor** and **Choreo Editor** in the same sentence. Better to stick to Web Editor for consistency. 
Also, Should we consider rephrasing to avoid repeating Web Editor?
 Something similar to:
“The Statement Editor in the Choreo Web Editor allows you to construct Ballerina statements depending on the use case you want to build. It is an intuitive visual editor that is easy to use even without an in-depth knowledge of the Ballerina Programming Language.” 
We can perhaps link off to ballerina.io from Ballerina Programming Language.

Reply:
Changed as suggested.